app/javascript/components/common/GraphicalIcon.tsx

```diff
@@ -5,8 +5,8 @@ export function GraphicalIcon({
   icon,
   className = '',
   category,
-  width = null,
-  height = null,
+  width = 0,
+  height = 0,
```

Comment:
This means something very different I think. It'll set the images to have 0px width/height if the thing isn't specified, rather than using the default of the image.

Reply:
You are right, had a proper look on it, and the correct default value is `undefined` if you want to get the default sizes. I updated the type signature on [GraphicalIcon.](https://github.com/exercism/website/blob/40e6f245bd3e35fa9c29d8e75034246936b1bf81/app/javascript/components/common/GraphicalIcon.tsx#L4-L10)